cfme/services/dashboard/ssui.py

```diff
@@ -57,6 +65,14 @@ def num_of_rows(self):
     return rows
 
 
+@Dashboard.results.external_implementation_for(ViaSSUI)
+def results(self):
```

Comment:
now that i take a closer look while readingthe pr backwards, i wonder what to do with results vs num_of_rows

i believe we should take a closer look at whats correct

Reply:
num_of_rows() and results() both are trying to validate the count of services displayed on the dashboard. I have wrapped them into one and not check according to versions in tests anymore .